---
arturl_encode: "68747470733a2f2f:626c6f672e6373646e2e6e65742f4d4d4d696e444444696e67:2f61727469636c652f64657461696c732f3934363232333531"
layout: post
title: "处理Python与Gurobi接口问题及解决办法"
date: 2024-07-26 14:25:51 +08:00
description: "将Gurobi安装到python库，先进入安装目录，在命令提示符中运行setup.py 中安装失败。"
keywords: "python安装gurobi拒绝访问"
categories: ['未分类']
tags: ['Setup', 'Python', 'I']
artid: "94622351"
image:
  path: https://api.vvhan.com/api/bing?rand=sj&artid=94622351
  alt: "处理Python与Gurobi接口问题及解决办法"
render_with_liquid: false
featuredImage: https://bing.ee123.net/img/rand?artid=94622351
featuredImagePreview: https://bing.ee123.net/img/rand?artid=94622351
---

# 处理Python与Gurobi接口问题及解决办法

###### 将Gurobi安装到python库，先进入安装目录， [在命令提示符中运行setup.py](http://xn--setup-4n1hu6bs1zv4f1z0b6h3cy2ejz3cu8v.py) 中安装失败。

当输入python
[setup.py](http://setup.py)
install报错：
  
C:\gurobi811\win64>python
[setup.py](http://setup.py)
install
  
running install
  
running build
  
running build\_py
  
creating build
  
creating build\lib
  
creating build\lib\gurobipy
  
copying python37\lib\gurobipy\_
*init*
\_.py -> build\lib\gurobipy
  
copying python37\lib\gurobipy\gurobipy.pyd -> build\lib\gurobipy
  
running install\_lib
  
creating C:\Program Files (x86)\Python37-32\Lib\site-packages\gurobipy
  
error: could not create ‘C:\Program Files (x86)\Python37-32\Lib\site-packages\gurobipy’: 拒绝访问。
  
![报错在这里插入图片描述](https://i-blog.csdnimg.cn/blog_migrate/ecdf482c462df300750d2301aecd0614.png)
  
解决方法如下：
  
python
[setup.py](http://setup.py)
install --user
  
在之后添加 --user即可解决路径拒绝访问的问题。
  
![在这里插入图片描述](https://i-blog.csdnimg.cn/blog_migrate/7b67e56ebd5e062de3fba8309f1c0849.png)